### PR TITLE
Adjust the definition of min_checkpoint_frequency cdc

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-11981

Small adjustment to the definition of the `min_checkpoint_frequency` option. 
Keywords highlighted in technical overview (that is linked to from edited definition above).

## Rendered preview:

https://deploy-preview-19232--cockroachdb-docs.netlify.app/docs/v24.3/create-changefeed.html#min-checkpoint-frequency

### CODE REVIEW

File: src/current/v23.1/create-changefeed.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ Option | Value | Description
...
... least the specified duration between sending progress updates for the ranges it is watching to the coordinator. This can help you control the flush frequency of higher latency sinks to achieve better throughput. However, more frequent checkpointing can increase CPU usage. If this is set to `0s`, a node will flush messages as long as the high-water mark has increased for the ranges that particular node is processing. If a changefeed is resumed, then `min_checkpoint_frequency` is the amount of time that changefeed will need to catch up. That is, it could emit [duplicate messages]({% link {{ page.version.version }}/changefeed-messages.md %}#duplicate-messages) during this time. <br><br>**Note:** [`resolved`](#resolved-option) messages will not be emitted more frequently than the configured `min_checkpoint_frequency` (but may be emitted less frequently). If you require `resolved` messages more frequently than `30s`, you must configure `min_checkpoint_frequency` to at least the desired `resolved` message frequency. For more details, refer to [Resolved message frequency]({% link {{ page.version.version }}/changefeed-messages.md %}#resolved-timestamp-frequency).<br><br>**Default:** `30s`
```
Comment (kathancox) [inline src/current/v23.1/create-changefeed.md:173]: A little hard to see, but the sentence that have changed in each version are from:

> Controls how often nodes flush their progress to the [coordinating changefeed node]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}). Changefeeds will wait for at least the specified duration before a flush to the sink.

To:

> Controls how often a node's changefeed [aggregator]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}) will flush their progress to the [coordinating changefeed node]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}). A node's changefeed aggregator will wait at least the specified duration between sending progress updates for the ranges it is watching to the coordinator.

File: src/current/v23.1/how-does-an-enterprise-changefeed-work.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ When an {{ site.data.products.enterprise }} changefeed is started on a node, tha
...
+Each node uses its _aggregator processors_ to send back checkpoint progress to the coordinator, which gathers this information to update the _high-water mark timestamp_. The high-water mark acts as a checkpoint for the changefeed’s job progress, and guarantees that all changes before (or at) the timestamp have been emitted. In the unlikely event that the changefeed’s coordinating node were to fail during the job, that role will move to a different node and the changefeed will restart from the last checkpoint. If restarted, the changefeed may [re-emit messages]({% link {{ page.version.version }}/changefeed-messages.md %}#duplicate-messages) starting at the high-water mark time to the current time. Refer to [Ordering Guarantees]({% link {{ page.version.version }}/changefeed-messages.md %}#ordering-and-delivery-guarantees) for detail on CockroachDB's at-least-once-delivery-guarantee and how per-key message ordering is applied.
```
Comment (kathancox) [inline src/current/v23.1/how-does-an-enterprise-changefeed-work.md:14]: Just a small change to italicize key words on this technical overview page.

**Review by rharding6373 — APPROVED** [review]
LGTM! Thank you for clarifying this.

**Review by rmloveland — APPROVED** [review]
LGTM

**Pull request merged**